### PR TITLE
Autoencoder Imgs integration

### PR DESCRIPTION
**Pull request by lithomas1**
Goals.
1. Make autoencoder work with images
2. Get trained h5 file

### CODE REVIEW

**Comment by lithomas1** [comment]
Kento, can you figure out how to make it train. I think I have currently made a list with the dataset.

**Comment by KentoNishi** [comment]
I'm going to use Keras instead of TensorFlow!

**Comment by KentoNishi** [comment]
Current configuration of layers is like this:
image

**Comment by KentoNishi** [comment]
I doubled the dimensions for this because of rounding issues on the y axis. The original configuration was:
90 --> 45 --> 23 --> 46 --> 92
The output didn't match the input!
So I changed it to the following:
180 --> 90 --> 45 --> 90 --> 180

**Comment by KentoNishi** [comment]
@lithomas1 review before merging!

**Comment by lithomas1** [comment]
seems legit

**Comment by KentoNishi** [comment]
I have Japanese school today so I won't be able to do anything :P
